Enterprise-CMCS/managed-care-review
# MCR-4324: remove cms email settings flag

## Summary

[MCR-4324](https://jiraent.cms.gov/browse/MCR-4324)

**NOTE**: I have not removed state analysts from parameter store yet. Once this PR is merged and promoted, I will then remove them from all environment parameter store.

- Remove `read-write-state-assignments` flag from the codebase
- Remove `getStateAnalystsEmails` and  `getStateAnalystsSettings.ts` parameter store functions
- Remove `fetchEmailSettings` GraphQL query and resolver
- Update resolvers to only query state analysts from the database, removing querying from parameter store.
- Update the `Settings` component to only use `fetchMcReviewSettingsQuery` for querying email configurations and state analysts.
- Update Cypress tests and commands to only use `fetchMcReviewSettings` query, removing all instances of `fetchEmailSettings`

#### Related issues

#### Screenshots

#### Test cases covered

- No new tests:
   - Removed tests checking state analysts from parameter store.
  - Updated existing tests to query analysts from the database



## QA guidance

- I would check the workflows that would require emailing state assignments to make emails are still being send and contain state analysts from the DB.
